Source/qol/stash.cpp

```diff
@@ -607,4 +612,51 @@ void GoldWithdrawNewText(string_view text)
 	}
 }
 
+bool AutoPlaceItemInStash(Player &player, const Item &item, bool persistItem)

```

Comment:
Nice, this would also work nicely with a potential text (like the stores) version of the stash :)